.github/workflows/lint.yaml

```diff
@@ -102,3 +102,7 @@ jobs:
       - name: Check if Operator CRDs are up to date
         # We have to add the current directory as a safe directory or else git commands will not work as expected.
         run: git config --global --add safe.directory $(realpath .) && make crds-up-to-date
+
+      - name: Check if derived functions are up to date
+        # We have to add the current directory as a safe directory or else git commands will not work as expected.
+        run: git config --global --add safe.directory $(realpath .) && make derive-up-to-date
```

Comment:
Can't the checkout action be configured to mark the directory as safe? Or alternatively, could we use the teleport-checkout action instead?

Reply:
We can but this is a back port and the original PR isn’t mine